azurerm/data_source_builtin_policy_definition.go

```diff
@@ -0,0 +1,83 @@
+package azurerm
+
+import (
+	"fmt"
+
+	"github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2018-05-01/policy"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceArmBuiltInPolicyDefinition() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceArmBuiltInPolicyDefinitionRead,
+		Schema: map[string]*schema.Schema{
+			"display_name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"management_group_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"type": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
```

Comment:
Other properties i think we should export after looking at the resource are:
- `policy_type`
- `policy_rule`
- `metadata`
- `parameters`